add-domain

```diff
@@ -1,3 +1,4 @@
+reltio-apps.com
```

Comment:
IF I may give a suggestion, then add this record to `add-wildcard-domain` for better protection, as it appears the domain is only to be used for evil.

Reply:
Thanks for the suggestion, just added.

Reply:
So this issue can be closed?